Case Study/CaseStudy-Phase-4/WebContent/view/Js/Create.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * This javascript file contains all the code related to the Create.htmml file.
+ */
+
+function validate () {
+	var make = document.getElementById("make").value;
+	var model = document.getElementById("model").value;
+	var engine = document.getElementById("engine").value;
+	var fuel = document.getElementById("fuel").value;
+	var milage = document.getElementById("milage").value;
+	var price = document.getElementById("price").value;
+	var roadtax = document.getElementById("roadtax").value;
+	var onroad = document.getElementById("onroad").value;
+	var ac = document.getElementById("ac").value;
+	var steering = document.getElementById("steering").value;
+	var kit = document.getElementById("kit").value;
+	var count = true;
+	if(make == "") {
+		count = false;
+		document.getElementById("make-error").innerHTML = "Enter make";
+		return false;
```

Comment:
don't return from each validation.display all error messages every time instead of displaying one by one